packages/arcgis-rest-places/test/findPlacesNearPoint.test.ts

```diff
@@ -74,4 +74,22 @@ describe("findPlacesNearPoint()", () => {
     expect(nextPage.nextPage).toBeUndefined();
     expect(url).toContain("token=MOCK_KEY");
   });
+
+  it("verify endpoint", async () => {
+    fetchMock.mock("*", placeNearPointMockNoMoreResults);
+
+    const response = await findPlacesNearPoint({
+      x: -3.1883,
+      y: 55.9533,
+      radius: 10,
+      endpoint:
+        "https://placesdev-api.arcgis.com/arcgis/rest/services/places-service/v1/places/near-point",
```

Comment:
Similar to Patrick's comment above, I think all these example URLs in the test spec files should be changed to 
```suggestion
        "https://places-api.arcgis.com/arcgis/rest/services/places-service/v1/places/near-point",
```